WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -694,6 +727,8 @@ private extension ProductsViewController {
         }
         displayEmptyStateViewController(emptyStateViewController)
         emptyStateViewController.configure(config)
+        // Hide toolbar when user doesn't have products
+        showOrHideToolBar()
```

Comment:
nit: I this call belongs on the `didEnter` method in the `.noResultsPlaceholder` case. As this method is only for showing the no products overlay and nothing more!

Reply:
You're right. I've moved to a new function that is called in init method.